Documentation/PUBLIC_APIs.md

```diff
@@ -319,13 +320,84 @@ AEPDecisionScope* decisionScope2 = [[AEPDecisionScope alloc] initWithName: @"myS
 
 ---
 
+### updatePropositionsWithCompletionHandler
+
+This API dispatches an event for the Edge network extension to fetch decision propositions, for the provided decision scopes array, from the decisioning services enabled in the Experience Edge. The returned decision propositions are cached in-memory in the Optimize SDK extension and can be retrieved using `getPropositions` API.
+
+<InlineAlert variant="help" slots="text"/>
```

Comment:
Please check if we need InlineAlert tag in SDK doc, as this might not be reflected in SDK document separately. We can still keep the content as normal test

Reply:
Sure, Done.